### PR TITLE
Fix outdated cache configuration in s3 tests: s3_storage_policy_by_defau…

### PR DESCRIPTION
**Pull request by kssenii**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by vdimir** [comment]
> Conflicting files
> ...
> tests/integration/test_temporary_data_in_cache/test.py 


You may want just revert commit [`163b5a9` (#48424)](https://github.com/ClickHouse/ClickHouse/pull/48424/commits/163b5a97978a1cbfbecc6fc5fdfe873b1af58655) from current branch, the same change is in master

**Comment by robot-ch-test-poll2** [comment]
This is an automated comment for commit b00c8598383415dbc08494480d9517a08510f670 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/48424/b00c8598383415dbc08494480d9517a08510f670/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

**Comment by kssenii** [comment]
We either need to add a new log level below `test` or close this PR because all stateless tests cannot survive `test` level logging from cache and fail with `"Timeout exceeded (180 s) while flushing system log 'DB::TextLog'"`. Another solution is to remove verbose test logs from cache code, but it is already an important minimum, removing them will make it impossible to debug cache related stress test failures.

Actually most cache related stress tests failures happen with `test.hits` and `test.visits` (in stress test we force s3 storage for them, so stateful tests are run on top of s3 in stress test) and there we already correctly use `s3_cache` storage policy which correctly turns on the cache. 

@vdimir, may be already stateful tests with `test.hits` and `test.visits` have some queries with joins which use `temporary data in cache` feature and so it is enough for testing this feature?

**Comment by alexey-milovidov** [comment]
@kssenii it requires update.

**Comment by vdimir** [comment]
> @vdimir, may be already stateful tests with test.hits and test.visits which have queries with joins which use temporary data in cache feature and so it is enough for testing this feature?

More is better, however, if we can't enable it globally right now, I don't mind. I'll check what we have in test.hits and test.visits and add cases if needed

**Comment by kssenii** [comment]
Another solution is increase log level for stateless tests from `test` to `trace`. IMHO `test` log level is important for stress tests, but not for stateless tests, which can be reproduced and debugged locally unlike stress tests. @alexey-milovidov, is it ok to increase?

**Comment by alexey-milovidov** [comment]
Yes, ok for me.

**Review by vdimir — APPROVED** [review]

File: tests/config/install.sh [hunk 1]
```diff
@@ -43,7 +43,7 @@ ln -sf $SRC_PATH/config.d/transactions.xml $DEST_SERVER_PATH/config.d/
 ln -sf $SRC_PATH/config.d/encryption.xml $DEST_SERVER_PATH/config.d/
 ln -sf $SRC_PATH/config.d/CORS.xml $DEST_SERVER_PATH/config.d/
 ln -sf $SRC_PATH/config.d/zookeeper_log.xml $DEST_SERVER_PATH/config.d/
-ln -sf $SRC_PATH/config.d/logger_test.xml $DEST_SERVER_PATH/config.d/
+ln -sf $SRC_PATH/config.d/logger_trace.xml $DEST_SERVER_PATH/config.d/
```
Comment (azat) [inline tests/config/install.sh:46]: If that was the intention I guess `logger_test.xml` can be removed

**Pull request merged**